### PR TITLE
Isentropic Analysis Notebook

### PR DESCRIPTION
**Pull request by tjwixtrom**
Here is a first draft of a simple description and example of isentropic analysis. I kept this pretty basic, we can iterate  to go into more depth if desired, at this point additional input is probably best.

### CODE REVIEW

File: site/projects/isentropic_analysis.md [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
+[comment]: <> (![under construction]&#40;https://images2.minutemediacdn.com/image/upload/c_fit,f_auto,fl_lossy,q_auto,w_728/v1555999902/shape/mentalfloss/under_construction1_0.gif?itok=Pn9g_wu6&#41;)
 
 ## Description
 
-_description here_
+Create plots for using isentropic coordinates within synoptic analysis and forecasting.
 
 ## Related Training Notebooks
 
+[Isentropic Introduction](notebooks/visualization/isentropic_introduction.ipynb)
```
Comment (lesserwhirls) [inline site/projects/isentropic_analysis.md:12]: Looks like the build docs check is failing on this line. We've been referencing links to notebooks using nbviewer, but it's being a bit flaky at the moment...and is making me rethink that choice :-/

**Review by lesserwhirls — CHANGES_REQUESTED** [review]

File: site/projects/isentropic_analysis.md (hunk 1 above)
Comment (tjwixtrom) [reply site/projects/isentropic_analysis.md]: Hmm...I updated the link and there's still a problem. Seems to still fail at the same line, don't really know what the problem is.

**Comment by lesserwhirls** [comment]
Looks like we are having a similar issue with the links on nbviewer as we have on Unidata/pyaos-ams-2021#72. Not an error on your end, but more in the way we check for broken links.

**Review by lesserwhirls — APPROVED** [review]
Awesome - thank you @tjwixtrom!

**Pull request merged**